libraries/joomla/database/driver/mysqli.php

```diff
@@ -557,8 +557,7 @@ public function execute()
 
 		if (!is_object($this->connection))
 		{
-			JLog::add(JText::sprintf('JLIB_DATABASE_QUERY_FAILED', $this->errorNum, $this->errorMsg), JLog::ERROR, 'database');
-			throw new RuntimeException($this->errorMsg, $this->errorNum);
+			throw new RuntimeException('Could not connect to MySQL.');
```

Comment:
In some ways it's worse because if there was an error on a previous query it will actually show that (not sure if that's intended by code for when a query times out tho) Rather than the current query